api/routes/saved_jobs.py

```
@@ -0,0 +1,304 @@
+            .eq("job_id", job_id)
+            .execute()
+        )
+
+        if response.data:
+            logger.info(f"User {user_id} unsaved job {job_id}")
+            return {"message": "Job unsaved successfully"}
+        else:
+            # Job wasn't saved, but that's ok
+            return {"message": "Job was not saved"}
+
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)